.changeset/khaki-schools-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-node': patch
+---
+
+Only send cache header when static resource is 200
```

Comment:
this should probably have a prefix for consistency with the rest of the changelog: https://github.com/sveltejs/kit/blob/master/packages/adapter-node/CHANGELOG.md
```suggestion
fix: only send cache header when static resource is 200
```